src/pandablocks_ioc/_pvi.py

```diff
@@ -82,22 +82,24 @@ def add_data_capture_pvi_info(
 ):
     component = SignalRW(
         name=epics_to_pvi_name(data_capture_record_name),
-        pv=data_capture_record_name,
-        widget=ButtonPanel(actions={"Start": "1", "Stop": "0"}),
+        read_pv=data_capture_record_name,
```

Comment:
Need to do the `read_pv` too, or leave it out since `SignalRW` will default to using the `write_pv` if the `read_pv` is missing.